[Student][E2E][MBL-13006]: Initial attempt at Quizzes E2E test

joehoag opened the pull request:
Also includes some accessibility fixes.

Please LMK if you can think of any useful ways to beef up this E2E test.

instructure-cx commented:
Scan submitted to Checkmarx

instructure-cx commented:
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1005742&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

StephenBrough commented on automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt:
```diff
@@ -7,13 +7,18 @@ import android.view.View
 import android.webkit.WebView
```
Probably want a license in this file

joehoag commented:
@StephenBrough : Thanks for the feedback.  I'll push a commit addressing your suggestions (copyright notice + "beefing up") before merging these fixes.

joehoag commented:
Latest commit adds the copyright header.  It also starts the quiz via the Assignments tab and resumes/completes the quiz via the Quizzes tab.
